test/suites/app-service-configurable/main_test.go

```diff
@@ -8,9 +8,13 @@ import (
 	"time"
 )
 
-const ascSnap = "edgex-app-service-configurable"
-const ascService = "edgex-app-service-configurable.app-service-configurable"
-const defaultProfile = "rules-engine"
+const (
+	ascSnap                   = "edgex-app-service-configurable"
+	appName                   = "app-service-configurable"
+	ascService                = ascSnap + "." + appName
+	defaultProfile            = "rules-engine"
+	appSrviceRulesServicePort = "59701"
```

Comment:
Port is changed from 59720 to [59701](https://github.com/edgexfoundry/app-service-configurable/blob/1a526f114784ef26915ff80fd995ea89bcf8e607/res/rules-engine/configuration.toml#L38). How was this passing before?

Reply:
There was an [extra `set`](https://github.com/canonical/edgex-snap-testing/blob/main/test/suites/app-service-configurable/config_test.go#L24) that misled the test. The extra `set` was added after `unset` because `unset` didn't bring the default port back, and the new port was set in Consul. This has been [fixed](https://github.com/canonical/edgex-snap-testing/pull/52/files#L53-L56) now by starting the service at the beginning, so Consul has the default port, and `unset+set` is no needed anymore.